frappe/templates/emails/new_message.html

```diff
@@ -1,42 +1,42 @@
 <!-- Include header part -->
-    <tr>
-        <td class="v-spacer"></td>
-        <td>
-            <table width="100%" border="0" cellpadding="0" cellspacing="0">
-                <tr>
-                    <td class="h-spacer"></td>
-                </tr>
-                <tr>
-                	<td>
-            			<h3>New Message</h3>
-            		</td>
-            	</tr>
-            	<tr>
-            		<td class="h-spacer"></td>
-            	</tr>
-            	<tr>
-                    <td>
-                    	<p>You have a new message from: <b>{{ from }}</b></p>
-                    </td>
-            	</tr>
-            	<tr>
-                    <td>
-            			<p>{{ message }}</p>
-                    </td>
-            	</tr>
-            	<tr>
-                    <td class="h-spacer">
-                    </td>
-            	</tr>
-            	<tr>
-                    <td>
-            			<p><a style="color: #7575ff;text-decoration: none;" href="{{ link }}">Login and view in Browser</a></p>
-                    </td>
-                </tr>
-            </table>
-        </td>
-        <td class="v-spacer" style="width: 20px; vertical-align: top;"></td>
-    </tr>
+	<tr>
+		<td class="v-spacer"></td>
+		<td>
+			<table width="100%" border="0" cellpadding="0" cellspacing="0">
+				<tr>
+					<td class="h-spacer"></td>
+				</tr>
+				<tr>
+					<td>
+						<h3>New Message</h3>
```

Comment:
it should be possible to translate {{_("New Message") }} please check all your changes